frontend/src/components/Modals/MangeWorkspace/Documents/Directory/FileRow/index.jsx

```diff
@@ -13,7 +13,6 @@ export default function FileRow({
   folderName,
   selected,
   toggleSelection,
-  expanded,
```

Comment:
if we dont need this prop, lets remove it from the Component when called and etc. We should not pass props that are not used, especially since we are not in TS and it wont be caught. I dont see any LOC that shows us doing this